Crate compilation broken on Android

Hello,

I've not been able to compile the latest release (0.11.2) of this crate on Android. Compilation fails with the following output:

```
   Compiling android_logger v0.11.2
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/android_logger-0.11.2/src/lib.rs:295:9
    |
295 |           Self::new_with_priority(match level {
    |  _________^^^^^^^^^^^^^^^^^^^^^^^-
296 | |             Level::Warn => LogPriority::WARN,
297 | |             Level::Info => LogPriority::INFO,
298 | |             Level::Debug => LogPriority::DEBUG,
299 | |             Level::Error => LogPriority::ERROR,
300 | |             Level::Trace => LogPriority::VERBOSE,
301 | |         })
    | |__________- an argument of type `&CStr` is missing
    |
note: associated function defined here
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/android_logger-0.11.2/src/lib.rs:282:12
    |
282 |     pub fn new_with_priority(priority: log_ffi::LogPriority, tag: &CStr) -> PlatformLogWriter {
    |            ^^^^^^^^^^^^^^^^^ ------------------------------  ----------
help: provide the argument
    |
295 ~         Self::new_with_priority(match level {
296 +             Level::Warn => LogPriority::WARN,
297 +             Level::Info => LogPriority::INFO,
298 +             Level::Debug => LogPriority::DEBUG,
299 +             Level::Error => LogPriority::ERROR,
300 +             Level::Trace => LogPriority::VERBOSE,
301 +         }, /* &CStr */)
    |
```

As specified in the output, it looks like function `new` doesn't pass the mandatory `tag` argument to `new_with_priority`: https://github.com/Nercury/android_logger-rs/blob/38186ece1056d90b8f75fd2a5eb5c860e0a1704e/src/lib.rs#L281-L302

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -28,26 +28,12 @@ jobs:
           - i686-linux-android
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Install Rust ${{ matrix.toolchain }}
-        uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@${{ matrix.toolchain }}
         with:
-          profile: minimal
-          toolchain: ${{ matrix.toolchain }}
           target: ${{ matrix.target }}
 
-      - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-
-      - name: Generate docs
-        uses: actions-rs/cargo@v1
-        with:
-          command: doc
-
-      - name: Run tests
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - run: cargo build --target=${{ matrix.target }}
+      - run: cargo doc --target=${{ matrix.target }}
+      - run: cargo test --target=${{ matrix.target }}

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -292,13 +292,16 @@ impl<'a> PlatformLogWriter<'a> {
 
     #[cfg(target_os = "android")]
     pub fn new(level: Level, tag: &CStr) -> PlatformLogWriter {
-        Self::new_with_priority(match level {
-            Level::Warn => LogPriority::WARN,
-            Level::Info => LogPriority::INFO,
-            Level::Debug => LogPriority::DEBUG,
-            Level::Error => LogPriority::ERROR,
-            Level::Trace => LogPriority::VERBOSE,
-        })
+        Self::new_with_priority(
+            match level {
+                Level::Warn => LogPriority::WARN,
+                Level::Info => LogPriority::INFO,
+                Level::Debug => LogPriority::DEBUG,
+                Level::Error => LogPriority::ERROR,
+                Level::Trace => LogPriority::VERBOSE,
+            },
+            tag,
+        )
     }
 
     #[cfg(not(target_os = "android"))]

